team/bundles/org.eclipse.compare/compare/org/eclipse/compare/structuremergeviewer/DiffTreeViewer.java

```diff
@@ -670,6 +673,35 @@ private void internalSetSelection(TreeItem ti, boolean fireOpen) {
 		}
 	}
 
+	private void expandCollapseAction(DoubleClickEvent event) {
```

Comment:
What is the purpose of this change?

Reply:
I have deleted that sorry for that.
I hate github....

Reply:
This is the next issue: https://github.com/eclipse-platform/eclipse.platform.ui/issues/1838 
There i create a new pull request.